tests/jquery/run_jquery.py

```diff
@@ -103,7 +106,7 @@ def run_servo(servo_exe, module):
             break
         line = line.rstrip()
         try:
-            name, test_result = parse_line_to_result(line)
+            name, test_result = parse_line_to_result(line.decode('utf-8'))
```

Comment:
I added this.  ```parse_line_to_result``` was being called with ```bytes``` in this context

Reply:
I don't think it should effect Python 2 -- but hopefully the CI would catch that?  I cannot use Python 2 locally since Fedora is missing the ```python2-virtualenv``` package.